Sources/ASN1Kit/ASN1Object.swift

```diff
@@ -37,6 +37,9 @@ public protocol ASN1Object {
 
     /// Whether the object is a constructed object.
     var constructed: Bool { get }
+    
+    /// The original encoding
+    var save: Data? { get set }
```

Comment:
This property should be gettable only. Maybe also rename

```suggestion
    /// The original encoding data the object was decoded from
    var originalEncoding: Data? { get }
```

Reply:
Making it `{ get }` only will require updating a bunch of internal APIs to use `ASN1Primitive` instead of `ASN1Object`, just FYI.

Reply:
... but I can reduce the visibility of some functions (in any case `ASN1Primitive` is not exported, so no public API changes here).

Reply:
nicely solved